query-node/mappings/src/council.ts

```diff
@@ -433,11 +436,15 @@ export async function council_NewCouncilElected({ event, store }: EventContext &
   // get election round and its candidates
   const electionRound = await getCurrentElectionRound(store)
 
-  // TODO: uncomment when following query will be working (after some QN patches make it to Olympia)
-  // const electedCandidates = await store.getMany(Candidate, { where: { electionRoundId: electionRound.id, member: { id_in: electedMemberIds } } })
-  const electedCandidates = (
-    await store.getMany(Candidate, { where: { electionRoundId: electionRound.id }, relations: ['member'] })
-  ).filter((item: Candidate) => electedMemberIds.find((tmpId) => tmpId === item.member.id.toString()))
+  const where = { electionRoundId: electionRound.id, status_json: { isTypeOf_eq: 'CandidacyStatusActive' } }
+  const candidates = await store.getMany(Candidate, { where })
+
+  const electedCandidates = candidates.filter((candidate) => electedMemberIds.includes(candidate.member.id))
+
+  // Set candidates elected status
+  electedCandidates.forEach((candidate) => {
+    candidate.status = new CandidacyStatusElected()
```

Comment:
I think you can put this under `councilMember.electedInCouncil = electedCouncil`